docs/functions/badge.ts

```diff
@@ -15,8 +15,8 @@ export default async (req: Request, context: Context) => {
   const pathParts = url.pathname.split("/").filter(Boolean); // remove empty parts
   const query = url.searchParams;
 
-  // Handle /badge-endpoint/config/:tag
-  if (pathParts.length === 3 && pathParts[0] === "badge-endpoint" && pathParts[1] === "config") {
+  // Handle /badge/config/:tag
```

Comment:
Do we know who's using this?

Reply:
Config just contains the JSON data that shields uses, and is not part of the public API